realm/realm-library/src/main/java/io/realm/internal/RowObject.java

```diff
@@ -0,0 +1,117 @@
+package io.realm.internal;
+
+import io.realm.ObjectChangeSet;
+import io.realm.RealmObjectChangeListener;
+
+/**
+ * Java wrapper for Object Store's {@code Object} class. Currently it is only used for object notifications.
+ */
+public class RowObject implements NativeObject {
```

Comment:
Just throwing ideas at the wall: `OsObject`, `NativeRealmObject`, `InternalObject`, `OsRealmObject` ... nothing is really great, so `RowObject` is probably fine for now.

But perhaps we should find a common prefix for all the ObjectStore classes?